### ISSUE TITLE
Undefined constant "SIGTERM"

### ISSUE THREAD

**Issue by nimafallahian** [symfony/symfony#51905]
### Symfony version(s) affected

6.3.5

### Description

On the latest release for symfony/messenger 6.3.5, this function has been added:
```
public function getSubscribedSignals(): array
    {
        return $this->signals ?? [\SIGTERM, \SIGINT];
    }
```
to file[messenger](https://github.com/symfony/messenger/tree/v6.3.5)/[Command](https://github.com/symfony/messenger/tree/v6.3.5/Command)
/ConsumeMessagesCommand.php. This causes the error Undefined constant "SIGTERM" on the envs without the library being installed by default. 

### How to reproduce

Run commands on systems without SIGTERM defined by default

**Comment by HypeMC** [comment]
Duplicate of: https://github.com/symfony/symfony/issues/51790, fixed in: https://github.com/symfony/symfony/pull/51791

**Comment by nimafallahian** [comment]
@HypeMC This issue is fixed on symfony/symfony repo, but the fix is not on symfony/messenger yet as it's still on 6.3.5 version.

**Comment by HypeMC** [comment]
@nimafallahian Hi, yes, the fix will be available in v6.3.6 once it's released.

**Comment by stof** [comment]
@nimafallahian actually, the fix is also in symfony/messenger if you install the dev version.
